app/views/pages/list-school-job.html.slim

```diff
@@ -0,0 +1,229 @@
+- content_for :page_title_prefix, "List a school job"
+
+.hiring-staff-landing-page
+  .govuk-grid-row.header class="govuk-!-margin-bottom-2"
+    .govuk-width-container
+      = govuk_phase_banner(classes: "govuk-!-margin-bottom-6", tag: { text: "beta" }) do
+        | This is a new service - #{govuk_link_to "your feedback", new_feedback_path} will help us to improve it.
+
+      .govuk-grid-column-two-thirds class="govuk-!-padding-left-0"
+        h1.govuk-heading-l = "Save time and money on recruitment"
+
+        p.govuk-body-l
+          | Teaching Vacancies is completely free. Save money on recruitment
+          |   advertising and spend it where it counts most - in the classroom.
+
+        p.govuk-body-l
+          | Teaching Vacancies puts you in control of your own recruitment. Post as
+          |   many vacancies as you want and directly publish and edit them for free.
+
+        .account-actions
+          = govuk_start_button(text: "Sign in", href: page_path("sign-in"))
+
+          .request-account-link
+            span.govuk-body-m class="govuk-!-margin-left-5"
+              = "or"
+            = govuk_link_to("request a Teaching Vacancies account", page_path("dsi-account-request"), class: "govuk-body-m govuk-!-margin-left-1")
+
+      .govuk-grid-column-one-third class="govuk-!-padding-left-0"
+        img class="govuk-!-margin-top-4" src=asset_pack_path("media/images/hiring_staff_landing_page/header_icons.png")
```

Comment:
the images should have dimensions (and alt tags @david-mears-dfe already mentioned) else the browser is going display them at original size and this is partly why you are going to have issues on mobile - see screen shot below